packages/common/src/ui/layout/tables/columns/EditableQuantityColumn.tsx

```diff
@@ -0,0 +1,73 @@
+import { ColumnDefinition } from './types';
+import { DomainObject } from '../../../../types';
+import { TextField, InputAdornment, Tooltip, Typography } from '@mui/material';
+import React, { useState, useEffect } from 'react';
+
+interface SomeQuantityEntity extends DomainObject {
+  quantity: number;
+  setQuantity: (rowKey: number, newQuantity: number) => void;
+}
```

Comment:
this is pretty cool and makes sure the row data has the required fields if you want to use this column